cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -122,6 +122,7 @@ import qualified System.Win32 as Win32
 
 data SetupScriptOptions = SetupScriptOptions {
     useCabalVersion          :: VersionRange,
+    useCabalSpecVersion      :: Maybe Version,
```

Comment:
What is this? Needs a comment, or SetupScriptOptions needs a reference to where these are defined. Ditto with useVersionMacros.